keel-bom/keel-bom.gradle.kts

```diff
@@ -0,0 +1,50 @@
+plugins {
+  `java-platform`
+  `maven-publish`
+}
+
+// without this building the pom fails when using the Nebula publishing plugin
+configurations {
+  create("compileOnly")
+}
+
+javaPlatform {
+  allowDependencies()
+}
+
+if (property("enablePublishing") == "true") {
+  publishing {
+    publications {
+      named<MavenPublication>("nebula") {
+        from(components["javaPlatform"])
+      }
+    }
+  }
+}
+
+dependencies {
+  api(platform("com.netflix.spinnaker.kork:kork-bom:${property("korkVersion")}"))
```

Comment:
Version has to be specified here or it's missing in the generated bom and then Gradle can't parse it.